core/domain/topic_domain.py

```diff
@@ -423,18 +423,21 @@ def from_dict(cls, subtopic_dict: SubtopicDict) -> Subtopic:
         return subtopic
 
     @classmethod
-    def create_default_subtopic(cls, subtopic_id: int, title: str) -> Subtopic:
+    def create_default_subtopic(
+        cls, subtopic_id: int, title: str,
+        url_frag: str = 'url-frag') -> Subtopic:
```

Comment:
```suggestion
    def create_default_subtopic(
        cls, 
        subtopic_id: int, 
        title: str,
        url_frag: str = 'url-frag'
    ) -> Subtopic:
```

Not sure if this is the correct solution.

Reply:
Done. PTAL.